examples/angular/row-selection-signal/src/app/filter.ts

```diff
@@ -39,10 +39,10 @@ import type { Column, Table } from '@tanstack/angular-table'
   standalone: true,
   imports: [CommonModule],
 })
-export class FilterComponent<T> implements OnInit {
+export class FilterComponent<T extends RowData> implements OnInit {
```

Comment:
Might be changing these generics to include TFeatures since it effects what APIs can be used in the logic of this component.